Dockerfile

```diff
@@ -1,7 +1,13 @@
-# NAME: dclong/rustpython
-FROM dclong/rust
-# GIT: https://github.com/legendu-net/docker-rust.git
+FROM rust:1.66-alpine3.17 as builder
 
-ENV PATH=/root/.cargo/bin:$PATH
-RUN cargo install --git https://github.com/RustPython/RustPython \
-    && /scripts/sys/purge_cache.sh
+WORKDIR /rust-python
+
+RUN apk --no-cache add musl-dev
+RUN cargo install --git https://github.com/RustPython/RustPython --features freeze-stdlib --profile release
```

Comment:
`freeze-stdlib` adds standard libs into the binary, it's very useful

Reply:
Awesome! Good to know! I was exploring RustPython a little bit but never used it for any serious project.